docs/production/data-sync/git-sync.mdx

```diff
@@ -1,91 +1,16 @@
-        - `Sync on server start up`: If this field is checked, then Mage will sync with the Git repo when Mage is started.
-    </Step>
-    <Step title="Fill out git credentials">
-        <AccordionGroup>
-        <Accordion title="SSH" icon="terminal">
-        <Warning>
-        **SSH keys with passphrase are not supported**
-
-        The Git integration currently does not support repositories that require an SSH key with a passphrase.
-
-        You can, however, use the Mage terminal and the git CLI to perform your git actions.
-        You'll still need to generate your SSH keys as normal. You can start a ssh-agent in the terminal so that
-        you don't have to enter your SSH key passphrase every time you perform an action.
-
-        `eval $(ssh-agent -s) && ssh-add ~/.ssh/id_rsa`
-
-        </Warning>
-        <Steps>
-        <Step title="Set up your SSH key">
-        This will allow your machine to access GitHub. 
-        Check out this sections this [guide](/production/data-sync/git-integration#configure-git-in-mage) to set up your SSH key.
-        </Step>
-        <Step title="Fill out the remaining fields.">
-        Filling out these fields will allow Mage to recreate your git and SSH settings if they are deleted (i.e. if your container restarts):
-            - `Username`: Your git username. If you push to your remote repo, this username will show up next to your commits.
-            - `Email`: Your git email. Used for the same purpose as your username.
-            - `SSH public key in base64`, `SSH public key in base64`: Enter your SSH public and private key encoded in base64. You can follow the instructions in the field description if you want to Mage to save your SSH keys.
-        </Step>
-        </Steps>
-        </Accordion>
-
-        <Accordion title="HTTPS/Token" icon="coin">
-        <Steps>
-            <Step title="Create a Git access token if you haven't already">
-                - [GitHub](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token)
-                - [GitLab](https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token)
-            </Step>
-            <Step title="Fill out the remaining fields.">
-                - `Username`: Your git username. If you push to your remote repo, this username will show up next to your commits.
-                - `Email`: Your git email. Used for the same purpose as your username.
-                - `Access token`: Enter the Git access token that you created in step 4.
-            </Step>
-        </Steps>
-        </Accordion>
-        </AccordionGroup>
-    </Step>
-    <Step title="Save your Git settings">
-    </Step>
-</Steps>
+To enable Git Sync, first [configure Git in Mage](/development/git/configure), then navigate to the settings page and tick the `Git Sync` box. To configure git settings, see [this guide](/production/data-sync/git-integration).
```

Comment:
What's the point of the "To configure git settings..." part here? should we remove that line since we added the dedicated configure page?

Reply:
Yep, nice catch